pkg/mkimage-raw-efi/initramfs-init

```diff
@@ -470,6 +470,37 @@ fi
 # zpool reports /dev/zfs missing if it can't read /etc/mtab
 ln -s /proc/mounts /etc/mtab
 
+# check if setup_loops=... was set
+if [ -n "$KOPT_setup_loops" ]; then
+	# locate boot media and mount it
+	ebegin "Mounting boot media"
+	nlplug-findfs $cryptopts -p /sbin/mdev ${KOPT_debug_init:+-d} \
+		${KOPT_usbdelay:+-t $(( $KOPT_usbdelay * 1000 ))} \
+		-n -r /tmp/rootfs
+	eend $?
+
+        # now add loops
+	ebegin "Setting up loopback block devices"
+	for l in $(cat /tmp/rootfs); do
+		losetup -r -f "$l"
+	done
+	eend $?
+fi
+
+# check if eve_installer=... was set
+if [ -n "$KOPT_eve_installer" ]; then
+        [ -n "$KOPT_debug_init" ] && export DEBUG=yes
+	ebegin "Lauhcning EVE installer."
```

Comment:
I suspect you mean "Launching" :-)

Reply:
Yes indeed. Fixed now. Thanks for spotting this @deitch